src/reports.ts

```diff
@@ -0,0 +1,34 @@
+import { PaginatedResponse } from './common';
+import request from './util/request';
+
+export type GetMeetingParticipantReportsParams = {
+  page_size?: number
+  next_page_token?: string
+  include_fields?: 'registrant_id'
+};
+
+export interface GetMeetingParticipantReportsResponse extends PaginatedResponse {
```

Comment:
Please change this to a `type` to match the existing convention.

Reply:
Done!